examples/interpreter-demo/serialize.js

```diff
@@ -17,10 +17,21 @@ var NODE_LOC_CONSTRUCTOR;
 var LINE_LOC_CONSTRUCTOR;
 
 /**
- * All non-primitives in the interpreter.
+ * All non-primitives in the interpreter as an Array.
  * @type {!Array<!Object>}
  */
- var objectList = [];
+var objectList = [];
+
+/**
+ * All non-primitves in the interpreter as a Set.
+ * Double the speed of serialization if ES6's Set is available.
+ * @type {Set|undefined}
```

Comment:
Looks like a `Map` to me.
```suggestion
 * A Map mapping non-primitves their corresponding indexes in objectList.
 * Doubles the speed of serialisation if ES6's Map is available.
 * @type {Map|undefined}
```

Reply:
Done.